### PR TITLE
Feat: [Term Entry] PyTorch Tensor Operations .narrow()

### LINKED ISSUE #5469: [Term Entry] PyTorch Tensor Operations .narrow()
### Reason/inspiration (optional)

We would like a new entry on the `.narrow()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensor-operations/terms/narrow/narrow.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by SaiTeja-002**
### Description

I've added a new entry for the `.narrow()` function in PyTorch at `docs/content/pytorch/concepts/tensor-operations/terms/narrow/narrow.md.`

### Issue Solved

Closes #5469 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensor-operations/terms/narrow/narrow.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+Title: '.narrow()'
+Description: 'Selects a narrow subsection of a tensor along the specified dimension.'
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/narrow/narrow.md]:
```suggestion
Description: 'Returns a narrow subsection of a tensor along the specified dimension.'
```

File: content/pytorch/concepts/tensor-operations/terms/narrow/narrow.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  - 'Data Science'
+Tags:
+  - 'Python'
+  - 'Machine Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/computer-science'
+---
+
+In PyTorch, the **`.narrow()`** function is used to select a subsection of a tensor along a specified dimension. It returns a narrowed view without copying data. It is useful for extracting specific sections of large tensors without creating new memory allocations.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/narrow/narrow.md]:
```suggestion
In PyTorch, the **`.narrow()`** method selects a subsection of a tensor along a specified dimension. It returns a narrowed view without copying the underlying data, making it efficient for extracting specific sections of large tensors without incurring additional memory allocations.
```

**Review by mamtawardhani — APPROVED** [review]
thank you for contributing to Codecademy Docs @SaiTeja-002, the entry is well written! 😄 

The entry looks good for a second round of review! 🚀

**Review by Sriparno08 — APPROVED** [review]
Looks good, @SaiTeja-002!

**Pull request merged**